app.py

```diff
@@ -101,6 +101,32 @@ def __del__(self):
         dpg.destroy_context()
 
 
+class Entity:
```

Comment:
Everything looks right, we will check everything when connected elements will be ready.